packages/devextreme/js/__internal/ui/html_editor/m_html_editor.ts

```diff
@@ -140,37 +167,39 @@ class HtmlEditor extends Editor<Properties> {
     super._initTemplates();
   }
 
-  _focusTarget() {
+  _focusTarget(): dxElementWrapper {
     return this._getContent();
   }
 
   _getContent(): dxElementWrapper {
     return this.$element().find(`.${HTML_EDITOR_CONTENT_CLASS}`);
   }
 
-  // @ts-expect-error ts-error
-  _focusInHandler({ relatedTarget }): void {
+  _focusInHandler(e: DxEvent): void {
+    const { relatedTarget } = e as DxEvent & { relatedTarget?: Element };
+
     if (this._shouldSkipFocusEvent(relatedTarget)) {
       return;
     }
 
     this._toggleFocusClass(true, this.$element());
-    // @ts-expect-error ts-error
-    super._focusInHandler.apply(this, arguments);
+
+    super._focusInHandler(e);
   }
 
-  // @ts-expect-error ts-error
-  _focusOutHandler({ relatedTarget }): void {
+  _focusOutHandler(e: DxEvent): void {
+    const { relatedTarget } = e as DxEvent & { relatedTarget?: Element };
+
     if (this._shouldSkipFocusEvent(relatedTarget)) {
       return;
     }
 
     this._toggleFocusClass(false, this.$element());
-    // @ts-expect-error ts-error
-    super._focusOutHandler.apply(this, arguments);
+
+    super._focusOutHandler(e);
   }
 
-  _shouldSkipFocusEvent(relatedTarget): boolean {
+  _shouldSkipFocusEvent(relatedTarget: Element | undefined): boolean {
```

Comment:
```suggestion
  _shouldSkipFocusEvent(relatedTarget?: Element): boolean {
```

Can we replace like this?